commands/meme.js

```diff
@@ -41,5 +41,35 @@ const Module = new Augur.Module()
 
             msg.channel.send({ files: [meme] });
         },
-    });
+    }).addCommand({
+        name: "Behold",
+        description: "creates a doofenshritz behold meme, with the arguments becoming your bottom text. Small chance of fire.",
+        permissions: (msg) => msg.channel.id == "121755900313731074" && msg.channel.permissionsFor(msg.member).has(["ATTACH_FILES", "EMBED_LINKS"]),
+        process: (msg) => {
+            function randomNumber(min, max) {
+                return Math.round(Math.random() * (max - min) + min);
```

Comment:
As written, `min` and `max` will only return with half the frequency as the mid numbers.
```suggestion
                let high = Math.max(max, min);
                let low = Math.min(max, min);
                return Math.floor(Math.random() * (high - low + 1)) + low;
```